configuration/uart.md

```diff
@@ -1,99 +1,96 @@
 # UART configuration
 
-There are two types of UART available on the Raspberry Pi -  [PL011](http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.ddi0183g/index.html) and mini UART. The PL011 is a capable, broadly 16550-compatible UART, while the mini UART has a reduced feature set.
+## Hardware
 
-All UARTs on the Raspberry Pi are 3.3V only - damage will occur if they are connected to 5V systems. An adaptor can be used to connect to 5V systems. Alternatively, low-cost USB to 3.3V serial adaptors are available from various third parties.
+All Raspberry Pi computers have at least two UARTs. The Raspberry Pi 4 family (BCM2711) has six UARTs.
 
-## Pi Zero, 1, 2 and 3 - two UARTs
+There are two different types of UART devices available on the Raspberry Pi:
+* [PL011](http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.ddi0183g/index.html), patterned on the [16550 UART](https://en.wikipedia.org/wiki/16550_UART) chip.
+* mini UART, patterned on the [8250 UART](https://en.wikipedia.org/wiki/8250_UART) chip.
 
-The Raspberry Pi Zero, 1, 2, and 3 each contain two UARTs as follows:
+The naming and numbering of the UARTs can be confusing because they use different names and numbers in different contexts. These names from Broadcom are authoritative:
 
-| Name | Type |
-|------|------|
-|UART0 |PL011 |
-|UART1 |mini UART |
+| SoC | UART0 | UART1 | UART2 | UART3 | UART4 | UART5 |
+| --- | --- | --- | --- | --- | --- | --- |
+| BCM2835 | PL011 | mini UART |
+| BCM2836 | PL011 | mini UART |
+| BCM2837 | PL011 | mini UART |
+| BCM2711 | PL011 | mini UART | PL011 | PL011 | PL011 | PL011 |
 
-## Pi 4 - six UARTS
+Inside the Raspberry Pi, the UARTs are able to connect in several ways. UART0 and UART1 can connect to either the GPIO header, or the onboard Bluetooth module, if there is one. UARTs two through five can be enabled and disabled, but have no alternate connections.
 
-The Raspberry Pi 4 has four additional PL011s, which are disabled by default. The full list of UARTs on the Pi 4 is:
+See the [Raspberry Pi 4 Model B Datasheet](https://www.raspberrypi.org/documentation/hardware/raspberrypi/bcm2711/rpi_DATA_2711_1p0_preliminary.pdf) section _5.1.2 GPIO Alternate Functions_, for pinout details. Earlier models are similar but with only the first two UARTs.
```

Comment:
Just noticed that this is using an absolute `https://www.raspberrypi.org/documentation/...` URL. Could you update it to be a relative URL instead please?

Reply:
Done!